W09H01/src/test/pgdp/hashset/TupleHashSetTest.java

```diff
@@ -1,4 +1,4 @@
-package pgdp.hashset;
+package test.pgdp.hashset;
```

Comment:
Changing the package results in more work for earlier cloners. Everbody has to change the this line back, if he uses (sys)links (see README.md).

Reply:
Ok, I will!